### DISCUSSION TITLE
Ability to change font for plates

### DISCUSSION THREAD

**Discussion by cubixty** [CoffeSiberian/truck-tools#8] [Proposals]
Currently its only the Luxemborg Plate font (i think). If possible could you add the ability to change to fonts such as the UK? Thank you

**Comment by CoffeSiberian** [comment]
Hey there, it's not a bad idea to swap the plate fonts. But I have only one problem with this and that is the size of the text that could affect it (although I could play with the size inside the preview) I'll take it to check it and see how it looks like. If you have example screenshots of the size differences with the same text they are welcome

For the moment I leave it as an interesting idea, since there are other priorities at the moment. Like for example to find some licence plate for ATS where it doesn't exist due to some limitations of the game

**Reply by cubixty** [reply]
Thank you for the reply. Will look forward to whenever you decide to invest some time into adding the feature.
